spec/requests/user_request_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe 'Users API', type: :request do
+  describe 'GET /api/v1/users/:user_id' do
+    it 'returns user information' do
+      user = create(:user)
+      get "/api/v1/users/#{user.id}"
+      expect(response).to have_http_status(200)
+
+      data = JSON.parse(response.body)
```

Comment:
you don't like to symbolize names? I think it makes the code a bit more readable but not all that necessary.